From 552474a477989d0d3514f126b8e314dccc27878c Mon Sep 17 00:00:00 2001
From: Kamel Bouhara <kamel.bouhara@bootlin.com>
Date: Wed, 1 Jul 2020 08:58:24 +0200
Subject: [PATCH] mariadb FTBFS fix

From: https://sources.debian.org/patches/libodb-mysql/2.4.0-4/mariadb_FTBFS_fix.patch/

Description: fix FTBFS with recent MariaDB versions
Don't include internal an header, only the main one.
Bug-Debian: https://bugs.debian.org/919374
Forwarded: no
Author: Laszlo Boszormenyi (GCS) <gcs@debian.org>

Signed-off-by: Kamel Bouhara <kamel.bouhara@bootlin.com>
---
 odb/mysql/mysql-types.hxx | 4 ++--
 odb/mysql/version.hxx     | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/odb/mysql/mysql-types.hxx b/odb/mysql/mysql-types.hxx
index 1254ccd..0e8f9a8 100644
--- a/odb/mysql/mysql-types.hxx
+++ b/odb/mysql/mysql-types.hxx
@@ -11,9 +11,9 @@ typedef char my_bool;
 typedef struct st_mysql_bind MYSQL_BIND;
 
 #ifdef LIBODB_MYSQL_INCLUDE_SHORT
-#  include <mysql_time.h>
+#  include <mysql.h>
 #else
-#  include <mysql/mysql_time.h>
+#  include <mysql/mysql.h>
 #endif
 
 #endif // ODB_MYSQL_MYSQL_TYPES_HXX
diff --git a/odb/mysql/version.hxx b/odb/mysql/version.hxx
index 05b4ba5..e711c2e 100644
--- a/odb/mysql/version.hxx
+++ b/odb/mysql/version.hxx
@@ -12,7 +12,7 @@
 #ifdef LIBODB_MYSQL_INCLUDE_SHORT
 #  include <mysql_version.h>
 #else
-#  include <mysql/mysql_version.h>
+#  include <mysql/mysql.h>
 #endif
 
 #include <odb/version.hxx>
-- 
2.26.2

